planet.js

```diff
@@ -15,7 +15,7 @@ import {makePromise} from './util.js';
 // import * as THREE from './three.module.js';
 // import { makeTextMesh } from './vr-ui.js';
 
-const presenceHost = `wss://${document.location.hostname}:4443`;
+let presenceHost = '';
```

Comment:
Does this even need to be a variable up here anymore?

Reply:
fixed